src/twix.coffee

```diff
@@ -515,3 +515,6 @@ if  typeof(define) == "function"
   define "twix", ["moment"], (moment) -> makeTwix(moment)
 
 @Twix = makeTwix(@moment) if @moment?
+
+# Also checks globals (Meteor)
+@Twix = makeTwix(moment) if moment?
```

Comment:
I'm worried that it's always hitting that case, since Moment sets the global in the browser too. Perhaps the whole thing should look something like this:

```coffee
momentToUse = @moment || moment
@Twix = makeTwix(momentToUse)
locale(momentToUse, @Twix)
```